docs/fast_data/configuration/config_maps/erSchema.md

```diff
@@ -167,6 +171,140 @@ Example:
+:::
+
+In the _A -> B_ tab (and also the _B -> A_, which works the same) you can click on the _+_ Rule_ button to create a new rule like the normal ones but with two important differences:
+- you can decide which type of relationship between the two sides (instead of equality) based on the available [MongoDB comparison query operators](https://www.mongodb.com/docs/manual/reference/operator/query/#comparison);
+- the right side of the expression can be a field of the other projection or a constant value;
+
+You can also create a group of rules by clicking the _+ Group_ button. Based on [MongoDB logical query operators](https://www.mongodb.com/docs/manual/reference/operator/query/#logical), a group is a set of rules connected by the `and` (every rule in the group must be true) or `or` (at least one rule in the group must be true) logical operator.
+
+![Condition editor from pr_dishes to pr_reviews, A -> B tab](../../img/er-schema-unidirectional-rule-editor.png)
+
+:::tip
+The No Code feature has been implemented having in mind the idea to create ER Schemas following the structure of the Single View you want to create.
+
+Keep this in mind when adding projections and conditions, to get an ER Schema which is both functional and easy to understand.
+:::
+
+### Delete a projection or a condition
+
+You can always delete any projection or condition already configured. You have two ways of doing this.
+
+You can do it by selecting the projection box or the condition line you want to remove and pressing the _backspace_ or the _Canc_ key.
+
+Otherwise, you can select a projection or a condition and click on the _Delete_ button in the bottom right position of the right side panel.
+
+:::caution
+Removing a projection will also delete all of the conditions starting and ending from it.
+:::
+
+### Interact with the elements in the Canvas
+
+:::caution
+The position of the elements (boxes and lines) in the canvas is automatically calculated to give the best overview of the ER Schema. You are free to reorganize projections in the canvas but these updates will not be saved when you will commit your changes.
+:::
+
+The canvas is made to help you create and improve your ER Schema and also to give you a look at the whole picture of the Single View that will be created from it. To help you to have the full picture of it, there are several buttons on the bottom-left side of the canvas to adjust the zoom, to lock the canvas to move around without the risk of moving boxes and lines, and an _Autolayout_ button that restores the canvas elements to the best position for a complete view. 
+
+You can also zoom in and out with the scroll wheel of the mouse.
+
+Also, in case you make any mistake, you can use the _Undo_ and _Redo_ functionality: they can be toggled via the button at the bottom-right side of the canvas, or by pressing _Ctrl+Z__ and _Ctrl+Y_ (_Cmd+Z_ and _Cmd+Shift+Z_ for MacOS). 
```

Comment:
it should be Cmd+Y even for Mac. @fedepini can you confirm?

Reply:
Yes, it's CMD + Y, I would not include CMD + Shift + Z since it does not always work

Reply:
So, like this:

```suggestion
Also, in case you make any mistake, you can use the _Undo_ and _Redo_ functionality: they can be toggled via the button at the bottom-right side of the canvas, or by pressing _Ctrl+Z__ and _Ctrl+Y_. 
```